### PR TITLE
Fixes #7041: Add httpd log context to /var/log/rudder/apache2

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/7041

### CODE REVIEW

File: rudder-webapp/SOURCES/rudder-webapp.fc [hunk 1]
```diff
@@ -1,2 +1,3 @@
 /var/rudder/inventories/accepted-nodes-updates gen_context(system_u:object_r:httpd_sys_rw_content_t,s0)
-/var/rudder/inventories/incoming               gen_context(system_u:object_r:httpd_sys_rw_content_t,s0)
\ No newline at end of file
+/var/rudder/inventories/incoming               gen_context(system_u:object_r:httpd_sys_rw_content_t,s0)
+/var/log/rudder/apache2(/.*)?                  gen_context(system_u:object_r:httpd_log_t,s0)
```
Comment (peckpeck) [inline rudder-webapp/SOURCES/rudder-webapp.fc]: It it on purpose that there is no newline at the end of the file ?

Comment (amousset) [inline rudder-webapp/SOURCES/rudder-webapp.fc]: No, just added it.

Comment (Kegeruneku) [inline rudder-webapp/SOURCES/rudder-webapp.fc]: I asked the same :dancer:

**Comment by peckpeck** [comment]
Ok Good

**Pull request merged**